tests/tests.js

```diff
@@ -175,3 +175,12 @@ test.skip('works with very large lengths', function (t) {
 	t.end();
 });
 */
+
+test('returns the correct name when called with toString', function (t) {
```

Comment:
I do see that you're using `Function#toString` but this isn't really a reliable approach - it's better to skip the check in IE and directly check the `name` property.